insights/parsers/tests/test_ps.py

```diff
@@ -386,3 +388,8 @@ def test_ps_aexww():
     assert 'ROUTER_METRICS_TLS_CERT_FILE' in p.get_environ('/usr/bin/openshift-router')[0]
     assert 'RELOAD_INTERVAL' in p.get_environ('/usr/bin/openshift-router')[0]
     assert len(p.get_environ('/usr/bin/openshift-router')) == 2
```

Comment:
I believe it would be helpful to add a few tests here to check values in addition to the len tests.